Creating a Production Image.md

```diff
@@ -0,0 +1,76 @@
+![image](https://user-images.githubusercontent.com/2686765/158505778-2b6b17d0-35d4-4b9d-bdda-abfb36398215.png)
+
+For the disk image, select "Select or create custom storage" and click manage:
+![image](https://user-images.githubusercontent.com/2686765/158506424-951edd69-5409-47eb-bd2e-9808e37c5b18.png)
+
+On the next screen, select a directory where you would like the raw disk image to live (if none are shown, click the `+` in the bottom left and enter a path). Once you have highlighted the correct place, click the `+` next to **Volumes** to create a new disk.
+![image](https://user-images.githubusercontent.com/2686765/158506649-a59d2281-0820-4ae3-b55d-d40e5e26c05c.png)
+
+In the dialog, give your new disk image a name (I typically pick one that corresponds to the type of production device I'm creating. In the "Format" drop down, select "raw". In the dialog shown below, this will create a disk image called `VxMark.img` that can be flashed onto hardware using [vx-iso](https://github.com/votingworks/vx-iso).
+
+![image](https://user-images.githubusercontent.com/2686765/158506825-ca2c8ec0-24ed-4c59-9314-a7e4f294114d.png)
+
+Also make sure to up the disk size to at least 50G, and ensure that "Allocate entire volume now" is selected. 
+![image](https://user-images.githubusercontent.com/2686765/158507233-c25ada70-f4cc-4e19-91ca-f1c8cc5d1839.png)
+
+Finally, ensure the newly created volume is selected on the previous dialog and click "Choose volume". 
+![image](https://user-images.githubusercontent.com/2686765/158507435-fb736aff-9d1a-4b96-9f7a-ff6b306fff50.png)
+
+We want to customize the install ahead of time so our Debian install knows to create an EFI system partition. 
+![image](https://user-images.githubusercontent.com/2686765/158505957-694e4858-8d4d-4190-913b-a1b24fde34a1.png)
+
+On the first customization screen, make sure you have [`edk2-ovmf`](https://github.com/tianocore/edk2) installed. This will allow you to select Secure Boot-compatible virtual firmware. (I don't think SB-compatibility is strictly necessary, so it might be the case that just standard UEFI virtual firmware will work). 
+![image](https://user-images.githubusercontent.com/2686765/158506084-b01cd6b0-4d58-4f2f-a4fa-6b4671fd3299.png)
+
+Other than that, you should be good to create the virtual machine using Debian. Follow the instructions [here](https://github.com/votingworks/vxsuite-complete-system/blob/main/INSTALL.md) to create a production-ready, locked down image. Then, compress your disk image using `lz4` and copy the compressed image to the install stick created in the [vx-iso](https://github.com/votingworks/vx-iso) process, and you should be ready to flash to hardware!
+
+<h2>Troubleshooting</h2>
+<h3>Dropped into an EFI shell after install</h3>
+If you are dropped into an EFI shell after you've installed (or even before), this can be because the VM firmware isn't perfect at remembering what to boot. The EFI shell looks like this:
```

Comment:
yeah, I do wonder if it just didn't write to the firmware that it could boot Debian, since you didn't even see the file system in the EFI shell. I'm not sure what was going on there.